app/javascript/packs/commentModPage.js

```diff
@@ -48,49 +50,6 @@ async function updateMainReactions(reactableType, category, reactableId) {
 }
 
 // Experience-Level JS
-function clearExpLevels() {
-  Array.from(
-    document.getElementsByClassName('level-rating-button selected'),
-  ).forEach((el) => {
-    el.classList.remove('selected');
-  });
-}
-
-async function updateExperienceLevel(currentUserId, articleId, rating, group) {
```

Comment:
these are verbatim duplicates of the same functions in `actionPanel.js`, I imported the correct version in here